kubectl-plugin/pkg/cmd/create/create_cluster.go

```diff
@@ -161,72 +166,145 @@ func (options *CreateClusterOptions) Complete(cmd *cobra.Command, args []string)
 	return nil
 }
 
-func (options *CreateClusterOptions) Validate() error {
-	resourceFields := map[string]string{
-		"head-cpu":                 options.headCPU,
-		"head-gpu":                 options.headGPU,
-		"head-memory":              options.headMemory,
-		"head-ephemeral-storage":   options.headEphemeralStorage,
-		"worker-cpu":               options.workerCPU,
-		"worker-gpu":               options.workerGPU,
-		"worker-tpu":               options.workerTPU,
-		"worker-memory":            options.workerMemory,
-		"worker-ephemeral-storage": options.workerEphemeralStorage,
-	}
+func (options *CreateClusterOptions) Validate(cmd *cobra.Command) error {
+	if options.configFile != "" {
+		if err := flagsIncompatibleWithConfigFilePresent(cmd); err != nil {
```

Comment:
decided to error if user tries to use a mix of config file and CLI flag to keep it simple and avoid merging or override logic for now

Reply:
SGTM for this.